Mathlib/Data/Set/Finite/Parity.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2024 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+-/
+import Mathlib.Data.Set.Card
+import Mathlib.Data.Set.Finite
+
+/-!
+# Finite sets and parity of their cardinality
+
+This file contains lemmas concerning finite sets and parity of their cardinality.
+-/
+
+namespace Set.Finite
+
+variable {V : Type _} {s : Set V}
+
+lemma one_lt_ncard_of_nonempty_of_even (hs : Set.Finite s) (hn : Set.Nonempty s)
+    (he : Even (Nat.card s)) : 1 < s.ncard := by
```

Comment:
Why are you using `Nat.card` on sets?

Reply:
Because I standardized on it in my staging area, I guess using `ncard` makes more sense here, changed to that:)